cvat/apps/events/handlers.py

```diff
@@ -373,8 +373,8 @@ def handle_delete(scope, instance, store_in_deletion_cache=False, **kwargs):
         user_email=uemail,
     )
 
-def handle_annotations_change(instance, annotations, action, **kwargs):
-    _annotations = deepcopy(annotations)
+def handle_annotations_change(instance, annotations, action, should_deepcopy=True, **kwargs):
+    _annotations = deepcopy(annotations) if should_deepcopy else annotations
```

Comment:
In my opinion we are trying to put into event too many information in this function, maybe that is worthy of discussing